presidio-analyzer/presidio_analyzer/predefined_recognizers/in_passport_recognizer.py

```diff
@@ -0,0 +1,52 @@
+# from typing import Optional, List, Tuple
```

Comment:
This file can be removed if it's not done yet

Reply:
done